### ISSUE TITLE
Expose Logtash, OS, and JVM Information

### ISSUE THREAD

**Issue by mjtrangoni** [BonnierNews/logstash_exporter#11]
Hi @christoe ,

Would it be possible exposing this information as labels of a metric with value 1, like prometheus does with `logstash_exporter_build_info`? I think this would be a **_nice to have_**  information for a text field.

* Logstash Information (version, pipeline information) 
* OS Information (name, arch, version, available processors)
* JVM Infomation (vm_version, vm_vendor, vm_name)

See, 
```
root@hostname ~ # curl -s http://localhost:9600/_node | jq
{
  "host": "hostname",
  "version": "5.6.5",
  "http_address": "127.0.0.1:9600",
  "id": "XXX",
  "name": "hostname",
  "pipeline": {
    "workers": 1234,
    "batch_size": 1234,
    "batch_delay": 10,
    "config_reload_automatic": false,
    "config_reload_interval": 3,
    "dead_letter_queue_enabled": false,
    "id": "main"
  },
  "os": {
    "name": "Linux",
    "arch": "amd64",
    "version": "3.10.0-693.11.1.el7.x86_64",
    "available_processors": 64
  },
  "jvm": {
    "pid": 34059,
    "version": "1.8.0_151",
    "vm_version": "1.8.0_151",
    "vm_vendor": "Oracle Corporation",
    "vm_name": "OpenJDK 64-Bit Server VM",
    "start_time_in_millis": 1513864863107,
    "mem": {
      "heap_init_in_bytes": 1073741824,
      "heap_max_in_bytes": 4151836672,
      "non_heap_init_in_bytes": 2555904,
      "non_heap_max_in_bytes": 0
    },
    "gc_collectors": [
      "ParNew",
      "ConcurrentMarkSweep"
    ]
  }
}
```

Thanks!

**Comment by christoe** [comment]
Created a PR with the above changes. @mjtrangoni, care to have a look and see if it covers your use cases?

**Comment by mjtrangoni** [comment]
@christoe yup, it works! Thank you very much!
```
# HELP logstash_info_jvm A metric with a constant '1' value labeled by name, version and vendor of the JVM running Logstash.
# TYPE logstash_info_jvm counter
logstash_info_jvm{name="OpenJDK 64-Bit Server VM",vendor="Oracle Corporation",version="1.8.0_151"} 1
# HELP logstash_info_node A metric with a constant '1' value labeled by Logstash version.
# TYPE logstash_info_node counter
logstash_info_node{version="5.6.5"} 1
# HELP logstash_info_os A metric with a constant '1' value labeled by name, arch, version and available_processors to the OS running Logstash.
# TYPE logstash_info_os counter
logstash_info_os{arch="amd64",available_processors="64",name="Linux",version="3.10.0-693.11.1.el7.x86_64"} 1
```

**Comment by christoe** [comment]
Resolved in v0.1.2.
